script.js

```diff
@@ -0,0 +1,41 @@
+//object containing inputs and outputs arrays
+let chatInputsOutputs = {
+  inputs: ['Hello', 'How are you?', 'What is your favourite colour?'],
+  outputs: [
+    'Hi',
+    'Great!',
+    "I have so many favorites it's hard to choose one.",
+  ],
+};
+console.log(chatInputsOutputs);
+console.log(chatInputsOutputs.inputs);
+console.log(chatInputsOutputs.outputs);
+
+function reply() {
+  let question = document.getElementById('input').value.toLowerCase();
+
+  console.log(question);
+
+  // statement to answer the question based on the index of the question.
+  if (
```

Comment:
This works just fine, but you can simplify it.
Since your `question` variable is converted to lowercase on line 15, you don't need to continue to call `toLowerCase()` on this variable.  

Good job using `map`. You can save the result of `chatInputsOutputs.inputs.map(v => v.toLowerCase()` in a variable since you call this twice. It would also be easier to read.